robolectric/src/main/java/org/robolectric/internal/SdkConfig.java

```diff
@@ -81,7 +81,7 @@ private DependencyJar createDependency(String groupId, String artifactId, String
   private static String getRobolectricVersion() {
     try {
       final Properties properties = new Properties();
-      properties.load(SdkVersion.class.getClassLoader().getResourceAsStream("version.properties"));
+      properties.load(SdkVersion.class.getClassLoader().getResourceAsStream("robolectric.version.properties"));
```

Comment:
Makes no difference to me. Is the convention for the project to amend the commit with a force push, or to add a second commit?